content/security/committers.md

```diff
@@ -134,14 +139,14 @@ not have a dedicated security list)
 
     d. `oss-security@lists.openwall.com` ([subscription not required](http://oss-security.openwall.org/wiki/mailing-lists)).
 
+This is the first point that any information regarding the vulnerability is made public.
+
 ### Complete
 
 17. The project team updates the project's security pages.
 
-18. Set the status of the vulerability to 'READY' in the internal portal. This notifies the
-    security team, which will submit the information to the CVE project.
-
-    This is the first point that any information regarding the vulnerability is made public.
+18. Add the link to the public announcement on the mailinglist as a 'reference' of type `CONFIRM` in the CVE, and set the status of the vulerability to `READY`.
```

Comment:
The https://cveprocess.apache.org website does not let a user send security emails until the status of the vulnerability is `READY`. Assuming this is the long term design of the website, I think we might need to mention updating the vulnerability state in step 16.

Reply:
Aah, good observation, thanks - updated accordingly.